src/LinkDotNet.NCronJob/Configuration/StringExtensions.cs

```diff
@@ -0,0 +1,31 @@
+using System.Security.Cryptography;
+using System.Text;
+
+namespace LinkDotNet.NCronJob;
+
+internal static class StringExtensions
+{
+    public static string GenerateConsistentShortHash(this string input)
+    {
+        var hashBytes = SHA256.HashData(Encoding.UTF8.GetBytes(input));
+        return Convert.ToBase64String(hashBytes)[..8]
+            .RemoveChar('+')
+            .RemoveChar('/')
+            .RemoveChar('=');
+    }
+
+    public static string RemoveChar(this string input, char charToRemove)
```

Comment:
nitpick:

Wouldn't it make it easier to do that in one pass-through and let `RemoveChar` either allow to pass in `params char[] toRemove` or have no parameter and check for those three inside the method?

My point is that still 3 strings are created where only one would be needed.
If we want to go extreme:
```csharp
public static string RemoveChar(this ReadOnlySpan<char> input)
```

Reply:
Yes, a single-pass approach might seem simpler from the caller's perspective, but we still need to iterate over each character, there's no performance advantage—and regex is notably slow. Although it would shift away from being a 'String extension' using ReadOnlySpan<char> would be optimal for performance. I’ll make some tweaks to the implementation.

Reply:
Sorry what I meant is that you call your function three times - therefore you pass through the loop three times.
I probably would do this in one pass through and check for all of those three chars